zio-json/shared/src/test/scala/zio/json/CodecSpec.scala

```diff
@@ -117,10 +116,19 @@ object CodecSpec extends ZIOSpecDefault {
           assertTrue(Pascaled("").toJsonAST.toOption.get == pascaled.fromJson[Json].toOption.get) &&
           assertTrue(Cameled("").toJson == cameled) &&
           assertTrue(Cameled("").toJsonAST.toOption.get == cameled.fromJson[Json].toOption.get) &&
-          assertTrue(Custom("").toJson == indianaJones) &&
-          assertTrue(Custom("").toJsonAST.toOption.get == indianaJones.fromJson[Json].toOption.get) &&
           assertTrue(OverridesAlsoWork("", 0).toJson == overrides)
         },
+        test("key transformation - except native") {
+          // Problem in scala-native seems to be with implementation of package scala.scalanative.regex.Parser, in particular method `parsePerlFlags`.
```

Comment:
Any chance this was reported up to the `scala-native` project as a problem, or is that a TODO?

Reply:
Never mind, I see you did in the attached issues above.